pinot-controller/src/main/java/com/linkedin/pinot/controller/helix/core/PinotHelixResourceManager.java

```diff
@@ -1031,7 +1031,8 @@ public Schema getTableSchema(@Nonnull String tableName) {
    * @throws TableAlreadyExistsException for offline tables only if the table already exists
    */
   public void addTable(@Nonnull TableConfig tableConfig) throws IOException {
-    String tableNameWithType = tableConfig.getTableName();
+    final String tableName = tableConfig.getTableName();
```

Comment:
We prefer `tableNameWithType` when the table name has `_OFFLINE` or `_REALTIME` suffix. In this way, we can directly know that the variable contains the suffix.